GameDefinition/Content/CaveKind.hs

```diff
@@ -232,7 +234,7 @@ noise = rogue
                     , ("tiny staircase", 1) ]
   , cstairAllowed = [ ("closed lift", 50), ("open lift", 50)
                     , ("tiny lift", 1) ]
-  , cdesc         = ""
+  , cdesc         = "Several machines still function, processors whirring through routines scheduled by dead men."
```

Comment:
Amazing.

Reply:
And here I grok the comma, since the second sentence is subordinate to the first in both meaning and grammar. Masterful.